### PR TITLE
Updated the favicon used for reports

### PR DESCRIPTION
**Pull request by vifor2**
This is a PR for [this issue](https://github.com/nccgroup/ScoutSuite/issues/134).
The favicon is now in the shape of a cloud.

### CODE REVIEW

**Comment by vifor2** [comment]
image

File: ScoutSuite/output/data/html/ruleset-generator.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
...
...nCsDvJwrA7ycKwO8nCsDvJwrA7ycKwO8nCsDvJwrA7ycKwL0nCsA1JwrApCcKwP4nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrAxCcKwPMnCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwPonCsD1JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwP8nCsDfJwrAqycKwP8nCsD/JwrA/ycKwP8nCsD/JwrA/ycKwPcnCsDnJwrA2CcKwOcnCsD/JwrA/icKwPMnCsDhJwrAYScKwCMnCsCnJwrA8ScKwP8nCsD/JwrA/ycKwP8nCsD3JwrA2ScKwLQnCsB7JwrAoScKwJInCsBsJwrAOCcKwAMnCsAAJwrACScKwFAnCsDqJwrA/ycKwP8nCsD/JwrA/ycKwPsnCsDuJwrAqycKwHcnCsBYJwrAjCcKwDInCsAAAAAAACcKwAAnCsADJwrAdCcKwO0nCsD/JwrA+ScKwP0nCsD/JwrA/ycKwOwnCsCJJwrASCcKwCMnCsACJwrAAAAAAAAAAAAAJwrAACcKwAUnCsA3JwrAYScKwE8nCsCuJwrA/ycKwP8nCsD/JwrA+ScKwOonCsB9JwrAAScKwAAAAAAAAAAAAAAAAAAAAAAAAAAAACcKwAAnCsAAJwrAGScKwIknCsDWJwrA5ycKwNInCsB9JwrAEicKwAAnCsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACcKwAAnCsADJwrAGCcKwCYnCsAVJwrAAScKwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA//8AAP//AACAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAEAAMABAADgAQAA/gMAAP8HAAD//wAA//8AAA==" rel="icon" type="image/x-icon" />
```
Comment (Aboisier) [inline ScoutSuite/output/data/html/ruleset-generator.html]: Can we not hardcode a base64 string 😂

**Comment by vifor2** [comment]
> Can we not hardcode the favicon with a base64 string 😂

Hmmm I did it like that because the original creators must've had a reason to do it like that, I just don't see why, but yeah sure.

**Comment by j4v** [comment]
Yeah I don't remember if there was a reason to do it this way but would be worth moving it to the `inc-` folders so its easier to update.

**Pull request merged**